### PR TITLE
dotCMS/core#18531 Create new message service

### PR DESCRIPTION
**Pull request by hmoreras**
migration of components to use the new service

### CODE REVIEW

File: src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
     #listing
     (rowWasClicked)="editContentType($event)"
     [actionHeaderOptions]="actionHeaderOptions"
-    placeholder="i18nMessages['contenttypes.fieldname.structure.name']"
```
Comment (hmoreras) [inline src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.html:5]: useless property.

File: src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.spec.ts [hunk 2]
```diff
@@ -145,8 +145,8 @@ describe('DotEditPageResolver', () => {
             );
             dotPageStateServiceRequestPageSpy.and.returnValue(of(mock));
 
-            resolver.resolve(route).subscribe((state: DotPageRenderState) => {
-                expect(state).toEqual(mock);
+            resolver.resolve(route).subscribe((state: DotPageRenderState) => {;
```
Comment (hmoreras) [inline src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.spec.ts:148]: @fmontes please help me to double check this, according to the logic in the service should be null

File: src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.spec.ts [hunk 3]
```diff
@@ -173,7 +173,7 @@ describe('DotEditPageResolver', () => {
             dotPageStateServiceRequestPageSpy.and.returnValue(of(mock));
 
             resolver.resolve(route).subscribe((state: DotPageRenderState) => {
-                expect(state).toEqual(mock);
+                expect(state).toBeNull();
```
Comment (hmoreras) [inline src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.spec.ts:176]: @fmontes please help me to double check this, according to the logic in the service should be null

File: src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.spec.ts [hunk 4]
```diff
@@ -145,8 +145,8 @@ describe('DotEditPageResolver', () => {
             );
             dotPageStateServiceRequestPageSpy.and.returnValue(of(mock));
 
-            resolver.resolve(route).subscribe((state: DotPageRenderState) => {
-                expect(state).toEqual(mock);
+            resolver.resolve(route).subscribe((state: DotPageRenderState) => {;
+                expect(state).toBeNull();
```
Comment (hmoreras) [inline src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.spec.ts:149]: @fmontes please help me to double check this, according to the logic in the service should be null

File: src/app/portlets/dot-apps/dot-apps-configuration-detail/dot-apps-configuration-detail.module.ts [hunk 5]
```diff
@@ -17,7 +18,8 @@ import { DotAppsConfigurationHeaderModule } from '../dot-apps-configuration-head
         DotKeyValueModule,
         DotCopyButtonModule,
         DotAppsConfigurationHeaderModule,
-        DotAppsConfigurationDetailFormModule
+        DotAppsConfigurationDetailFormModule,
+        DotDirectivesModule
```
Comment (fmontes) [inline src/app/portlets/dot-apps/dot-apps-configuration-detail/dot-apps-configuration-detail.module.ts:22]: Can't this be global?

File: src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.module.ts [hunk 6]
```diff
@@ -5,9 +5,10 @@ import { DotAppsConfigurationHeaderComponent } from './dot-apps-configuration-he
 import { DotCopyButtonModule } from '@components/dot-copy-button/dot-copy-button.module';
 import { DotAvatarModule } from '@components/_common/dot-avatar/dot-avatar.module';
 import { NgxMdModule } from 'ngx-md';
+import { DotDirectivesModule } from '@shared/dot-directives.module';
 
 @NgModule({
-    imports: [CommonModule, DotAvatarModule, DotCopyButtonModule, NgxMdModule],
+    imports: [CommonModule, DotAvatarModule, DotCopyButtonModule, NgxMdModule, DotDirectivesModule],
```
Comment (fmontes) [inline src/app/portlets/dot-apps/dot-apps-configuration-header/dot-apps-configuration-header.module.ts:11]: I think we need to find a way to make this global.

File: src/app/portlets/dot-edit-page/content/components/dot-edit-page-state-controller/components/dot-edit-page-lock-info/dot-edit-page-lock-info.component.html [hunk 7]
```diff
@@ -3,9 +3,9 @@
     class="page-info__locked-by-message"
     *ngIf="show"
 >
-    {{ dotMessageService.get('editpage.toolbar.page.locked.by.user', pageState.page.lockedByName) }}
+    {{ 'editpage.toolbar.page.locked.by.user' | dm:[pageState.page.lockedByName]}}
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/content/components/dot-edit-page-state-controller/components/dot-edit-page-lock-info/dot-edit-page-lock-info.component.html:6]: what is the `:` for?

Comment (hmoreras) [reply src/app/portlets/dot-edit-page/content/components/dot-edit-page-state-controller/components/dot-edit-page-lock-info/dot-edit-page-lock-info.component.html:6]: is a parameter for the pipe. and it does the following, https://github.com/dotCMS/core-web/blob/master/src/app/api/services/dot-messages-service.ts#L118

**Pull request merged**